src/renderer/components/+preferences/preferences.tsx

```diff
@@ -130,7 +131,17 @@ export class Preferences extends React.Component {
         <small className="hint">
           Proxy is used only for non-cluster communication.
         </small>
-
+        <h2>Terminal Shell</h2>
+        <Input
+          theme="round-black"
+          placeholder={process.env.SHELL}
```

Comment:
Unless I am mistaken about this but the `PTYSHELL` should also be considered, right?

```suggestion
          placeholder={process.env.SHELL || process.env.PTYSHELL}
```